data/RBGV_2023/metadata.yml

```diff
@@ -0,0 +1,875 @@
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
+  replicates: .na
+  methods: The term used to describe the fruit breadth or thickness was stored in
+    the notes section for greater clarity e.g. fruit, achene, capsule. Fruits which
+    were spherical or round in shape were inferred as possessing the same dimensions
+    in other planes.
+- var_in: leaf_length
+  unit_in: mm
+  trait_name: leaf_length
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
+  replicates: .na
+  methods: The term used to describe the length of the leaf or leaf analogue was stored
+    in the notes section for greater clarity, e.g. lamina, leaflet, blade etc.
+- var_in: leaf_width
+  unit_in: mm
+  trait_name: leaf_width
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
+  replicates: .na
+  methods: The term used to describe the width of the leaf or leaf analogue was stored
+    in the notes section for greater clarity, e.g. lamina, leaflet, blade etc.
+- var_in: petiole_length
+  unit_in: mm
+  trait_name: petiole_length
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
+  replicates: .na
+  methods: This data records the length of the petiole as recorded in the taxon description.
+- var_in: petiole_width
+  unit_in: mm
+  trait_name: petiole_width
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
+  replicates: .na
+  methods: This data records the width of the petiole as recorded in the taxon description.
+- var_in: seed_length
+  unit_in: mm
+  trait_name: seed_length
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
+  replicates: .na
+  methods: This data records seed length as described in the taxon description. Seeds
+    which were spherical or round in shape were inferred as possessing the same dimensions
+    in other planes.
+- var_in: seed_width
+  unit_in: mm
+  trait_name: seed_width
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
+  replicates: .na
+  methods: This data records the seed width as described in the text. Seeds which
+    were spherical or round in shape were inferred as possessing the same dimensions
+    in other planes.
+- var_in: seed_breadth
+  unit_in: mm
+  trait_name: seed_height
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
+  replicates: .na
+  methods: This data records the seed breadth as described in the text. Seeds which
+    were spherical or round in shape were inferred as possessing the same dimensions
+    in other planes.
+- var_in: plant_height
+  unit_in: mm
+  trait_name: plant_height
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
+  replicates: .na
+  methods: Maximum plant vegetative height of a mature individual.
+substitutions:
+- trait_name: storage_organ
+  find: rhizome_tuberous
+  replace: rhizome_fleshy
```

Comment:
Fantastic! It seems like a long time ago we extracted this trait, you guys did a great job of redoing the category for "stroage organ".

Reply:
I had stem tuber and then changed to this, thinking it is a rhizome that is "tuberous" rather than a type of "tuber". And I think any (or at least most) rhizome produces adventitious roots, wouldn't it?

Reply:
Yeah I think you're right, 'rhizome' probably has more weight over 'tuberous'